packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -428,9 +493,23 @@ class AuroraPostgresClusterEngine extends ClusterEngineBase {
           majorVersion: version.auroraPostgresMajorVersion,
         }
         : undefined,
+      features: version?._features,
     });
   }
 
+  public bindToCluster(scope: core.Construct, options: ClusterEngineBindOptions): ClusterEngineConfig {
+    const config = super.bindToCluster(scope, options);
+    if (options.s3ImportRole && !(config.features?.s3Import)) {
+      throw new Error (`s3Import is not supported for engine: ${this.engineVersion?.fullVersion}. Use an engine version that supports the s3Import feature.`);
+    }
+    if (options.s3ExportRole && !(config.features?.s3Export)) {
+      throw new Error (`s3Export is not supported for engine: ${this.engineVersion?.fullVersion}. Use an engine version that supports the s3Export feature.`);
```

Comment:
```suggestion
      throw new Error(`s3Export is not supported for Postgres version: ${this.engineVersion?.fullVersion}. Use a version that supports the s3Export feature.`);
```

BTW, this will be a weird message for the unversioned engine... Do you need a default? Like `${this.engineVersion?.fullVersion ?? '<unversioned>'}` or something?

Reply:
I was going to go with major version but perhaps unversioned is better

Reply:
dropped this when adding support for unversioned - we're going to skip validation when the unversioned aurorapostgres is requested